src/components/PromoBanner.vue

```diff
@@ -11,19 +11,19 @@
 		</button>
 		<div class="promo-banner__body">
 			<div class="discount">
-				<div class="discount__percentage">{{ discount }}</div>
+				<div class="discount__percentage">{{ active_promotion.discount }}</div>
 				<div class="discount__off">OFF</div>
 			</div>
 			<div>
-				<div class="code">
+				<div class="code" @click="copyCode">
 					Use promo code
-					<div>{{ code }}</div>
+					<div>{{ active_promotion.code }}</div>
 				</div>
```

Comment:
Love the copy! Zullen we een hover color op de bg zetten ` $bg-neutral-5` en een copy icon erachter plaatsen?
```suggestion
					<div>
				   	{{ active_promotion.code }}
				   	<hk-icon icon="fas fa-copy" class="ml-1" />
				  </div>
				</div>
```